crates/bevy_animation/src/lib.rs

```diff
@@ -472,6 +494,21 @@ impl AnimationClip {
             .push(variable_curve);
     }
 
+    /// Add a untargeted [`AnimationEvent`] to this [`AnimationClip`].
```

Comment:
The [`AnimationEvent`] type that you're linking to is private, and has no docs. We should probably fix that.

Reply:
Ah yeah. It is intentionally private / an implementation detail. Ill switch this to `Event`